pkg/translate/service.go

```diff
@@ -68,12 +76,78 @@ func (t *TranslateServiceServer) ListServices(
 	ctx context.Context,
 	req *translatev1.ListServicesRequest,
 ) (*translatev1.ListServicesResponse, error) {
-	return &translatev1.ListServicesResponse{}, nil
+	services, err := t.repo.LoadServices(ctx)
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, err.Error())
+	}
+
+	resp := &translatev1.ListServicesResponse{
+		Services: make([]*translatev1.Service, 0, len(services)),
+	}
+
+	for _, v := range services {
```

Comment:
We can use a generic function for the slice transformation. Please check how it is done in the `tw` project - `transform/transform.go`.